README.md

```diff
@@ -1,27 +1,119 @@
+_What this community hopes to have accomplished in the future_
 
-* Were downloaded from DownSub and converted to .vtt via https://subtitletools.com/convert-to-vtt-online/
-* A list of srclang codes can be found here: https://www.footprintaddons.com/my-account/articles-news/282-language-codes-subtitle-tracks
+Researchers, scientists, and analysts share their batch data analysis workflows without technical barriers using an open standard.  Sharing workflows this way is a usual occurrence and seen as a typical way of working.  The workflows are complete and run on a variety of environments; and people re-use shared workflow descriptions and build new workflows from them. No vendor dominates the ecosystem
 
-### Autogenerated Subtitles
+<a id="mission-of-the-cwl-project"></a>
+### Mission of the CWL Project
+_How we plan to achieve our vision_
 
-* We can optionally downloads YouTube's auto-generated subtitles via https://downsub.com/?url=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3D86eY8xs-Vo8
+The CWL project supports open consensus-based standards for command line data analysis
+workflows and tools.
 
+Specifically, we support the
+* pre-standards process by providing a neutral place of convening to discuss, propose and
+test ideas about command-line tool based workflow standards and related topics
+* standardization process by stewarding the development and delivery of standards in
+accordance with the [Open Stand principles](https://open-stand.org/about-us/principles/).  
+* post-standards life cycle by (1) promoting the released standards, (2) developing and maintaining
+related training and tools, and by (3) tracking deficits and other post-standardization feedback.
+<a id="editing-the-site---general-flow"></a>
+
+<a id="editing-the-site---general-flow"></a>
+## Editing the Site - General Flow
+
+1. On `main`, run a `git fetch --all` and pull in any changes.
+2. Switch to `add-jekyll-site` and then merge the changes via `git merge main`
+3. Make sure you're making your changes on the `add-jekyll-site` (or your own branch, which you can merge to `add-jekyll-site`).
+4. Push to `add-jekyll-site` and then check that everything looks good. 
+5. If all is well, head over to the [main repo](https://github.com/common-workflow-language/cwl-website) and create a Pull Request.
+
+<a id="config-files--local-development"></a>
+### Config Files & Local Development
+
+The site uses `_config.yml` for the production site, and `_config_staging.yml` for the staging site (which inherits, then overrides `_config.yml`). For ease of local development, you may wish to use a 3rd config file, e.g. `_config_local.yml`. If so, please don't add it to the repo.
+
+My `_config_local.yml` currently looks like this:
+
+```yaml
+title: CWL Local Site
+url: "cwl.test"
+baseurl: ""
+port: 80
+```
+
+The command for this is: `bundle exec jekyll serve --config "_config.yml,_config_local.yml"`. (Note the lack of spaces between the two config files).
+
```

Comment:
```
$ bundle exec jekyll serve --config "_config.yml"
Could not locate Gemfile or .bundle/ directory
```

Reply:
For Debian, I had install the following packages: ruby-dev, ruby-bundler ; that should probably be mentioned somewhere

Reply:
Okay, I figured out that I have to run `bundle` commands from the `content` directory..